runtime/bajun/src/benchmarking/orml_vesting.rs

```diff
@@ -0,0 +1,119 @@
+// This file is part of Acala.
+
+// Copyright (C) 2020-2024 Acala Foundation.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use super::utils::{get_vesting_account, lookup_of_account, set_balance};
+use crate::{AccountId, Balance, Balances, BlockNumber, Runtime, System, Vesting, BAJUN};
+use frame_benchmarking::{account, whitelisted_caller};
+use frame_support::traits::{fungible::Inspect, Get};
+use frame_system::RawOrigin;
+use sp_std::prelude::*;
+
+use orml_benchmarking::runtime_benchmarks;
```

Comment:
Todo: use the `frame-benchmarking::benchmarks` macro here and rewrite the benchmarks.